src/Network/Network/help/Get-AzBastionShareableLink.md

```diff
@@ -0,0 +1,246 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
```

Comment:
```suggestion
online version: https://learn.microsoft.com/powershell/module/az.network/get-azbastionshareablelink
```

Please also do so for `New-AzBastionShareableLink.md` and `Remove-AzBastionShareableLink.md`